20/format/Dockerfile

```diff
@@ -0,0 +1,3 @@
+# Image specifically designed to run clang-format on OpenRCT2 source files.
+FROM alpine
+RUN apk add --no-cache python3 clang19-extra-tools
```

Comment:
There's clang20 already, can you update it here as well? The main repo can then switch from `19-format` to `20-format` (version match is pure coincidental) in a separate change if necessary

Reply:
Just a heads up, updating this to clang 20 caused new formatting issues to pop up, which probably need to be resolved in its own commit / PR.
https://github.com/OpenRCT2/OpenRCT2/actions/runs/16639115663/job/47085767032?pr=24734